scalding-core/src/main/scala/com/twitter/scalding/serialization/StringOrderedSerialization.scala

```diff
@@ -22,7 +22,67 @@ import scala.util.control.NonFatal
 
 import JavaStreamEnrichments._
 
+object StringOrderedSerialization {
+  final def binaryIntCompare(leftSize: Int, seekingLeft: InputStream, rightSize: Int, seekingRight: InputStream): Int = {
```

Comment:
what is the `Int` here? why not just `compareBinary` to match the terminology we normally use?